scripts/spectrum-vars.js

```diff
@@ -144,9 +144,9 @@ const foundVars = await findUsedVars();
 
 systems.forEach((system) => {
     const varsPackage = system === 'express' ? 'expressvars' : 'vars';
-    const varsPath = path
-        .dirname(import.meta.resolve(`@spectrum-css/${varsPackage}`))
-        .replace(/^file:/, '');
+    const varsPath = path.dirname(
```

Comment:
import.meta.resolve was also providing a double drive prefix to the resulting file path in windows only.